org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/symbol/DocumentSymbolMapper.xtend

```diff
@@ -96,14 +96,14 @@ class DocumentSymbolMapper {
 		}
 
 		protected def String getName(QualifiedName qualifiedName) {
-			return qualifiedName?.toString;
```

Comment:
should we then use org.eclipse.xtext.ide.server.symbol.DocumentSymbolMapper.DocumentSymbolDetailsProvider.getDetails(EObject) 
to give the qualified name by default